prober/tls.go

```diff
@@ -28,6 +28,23 @@ func getEarliestCertExpiry(state *tls.ConnectionState) time.Time {
 	return earliest
 }
 
+func getLatestVerifiedChainExpiry(state *tls.ConnectionState) time.Time {
+	latestVerifiedChainExpiry := time.Time{}
```

Comment:
What does this output if there's no chains? You'll probably need insecure_skip_verify to test this.